app/src/main/res/values/strings.xml

```diff
@@ -190,6 +190,11 @@
     <string name="up_next">Up next</string>
     <string name="x_selected">%1$d selected</string>
 
+    <string name="error_credential_validation">Field cannot be empty</string>
+    <string name="server_unreachable">Server unreachable</string>
+    <string name="authentication_failed">Authentication failed</string>
+    <string name="No_network_connection_available">No network connection available</string>
+
```

Comment:
The names shouldn't have capital letters, and if they're toasts I'd prefer to put them with the other toasts and make them full sentences.